internal/cli/opsmanager/admin/backupstore/admin_opts.go

```diff
@@ -60,3 +63,12 @@ func (opts *AdminOpts) NewBackupStore() *opsmngr.BackupStore {
 
 	return backupStore
 }
+
+func (opts *AdminOpts) NewFileSystemConfiguration() *opsmngr.FileSystemStoreConfiguration {
```

Comment:
[q] why are we adding this here? can't this be closer to the file system commands? do this need to be shared with other commands as well?

Reply:
This is used by the create and the update commands. I thought to put it there so that I don't have to define it two times (inside CreateOpts and UpdateOpts). Do you want me to move it back?

Reply:
you're adding a method that only a subset of the consumers of this struct will need, so the question still stands do this need to be shared with all the commands that mix this struct? or do the other commands need the new fields as well?
If no then this should not be here, you can, either duplicate this function in both create and update, and that's ok, or you can create. a new shared struct closer to the file system package that they only use